client/post-editor/controller.js

```diff
@@ -53,8 +52,8 @@ function determinePostType( context ) {
 	return context.params.type;
 }
 
-function renderEditor( context ) {
-	context.primary = React.createElement( PostEditor );
+function renderEditor( context, props ) {
+	context.primary = React.createElement( PostEditor, props );
```

Comment:
Is there anything else in the props object at this point? Perhaps we can be explicit on passing through just what we need, analyticsPath, and analyticsTitle